dashboard/app/views/levels/_lab2.html.haml

```diff
@@ -0,0 +1,8 @@
+#codeApp{style: "position: absolute; top: 60px; bottom: 10px; left: 10px; right: 10px"}
+  #lab2-container{style: "height: 100%"}
+
+- content_for(:head) do
+  %script{src: webpack_asset_path("js/#{js_locale}/music_locale.js")}
+  %script{src: webpack_asset_path("js/#{js_locale}/standaloneVideo_locale.js")}
```

Comment:
For a future change - maybe we get a list of `@lab2_apps` from the backend with all the lab2 apps we need to locales for, and loop through those? Would be good to have a list in one place so when we onboard a new app type, we can just add it there

Reply:
That's a good idea.  And we could actually just make it a loop in Ruby on this page, since it's server-rendered.